<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <let name="config_file"
       value="$(find-pkg-share asr_sdm_image_classifier)/config/img_classifier.param.yaml"/>

  <node
    pkg="asr_sdm_image_classifier"
    exec="img_classifier_node"
    name="img_classifier_node"
    namespace="perception"
    output="screen"
    emulate_tty="true">

    <param from="$(var config_file)" />

    <!-- 在线模式时图像输入与输出重映射 -->
    <remap from="~/input/image" to="/sensing/camera/front/image_raw" />
    <remap from="~/output/image" to="/perception/image_classification/image" />
    <remap from="~/output/label" to="/perception/image_classification/label" />
  </node>
</launch>
